test/unit_overapproximate.jl

```diff
@@ -349,6 +349,17 @@ for N in [Float64]
     X = Ball1(zeros(N, 3), N(1))
     Y = overapproximate(X, Zonotope, BoxDirections{N}(3))
     @test Y == Zonotope(N[0, 0, 0], N[1 0 0; 0 1 0; 0 0 1])
+    B = BallInf(zeros(N, 3), N(1))
+    H = convert(HPolytope, B)
+    Z = Zonotope(N[0, 0, 0], N[1 0 0; 0 1 0; 0 0 1])
+    @test overapproximate(H, Zonotope, BoxDirections, algorithm="vrep") == Z
+    @test overapproximate(H, Zonotope, BoxDirections, algorithm="cpa") == Z
+    V = convert(VPolytope, B)
+    @test overapproximate(V, Zonotope, BoxDirections, algorithm="vrep") == Z
+    @test overapproximate(V, Zonotope, BoxDirections, algorithm="cpa") == Z
+    # test dispatch of internal functions
+    @test Approximations._overapproximate_zonotope_vrep(V, BoxDirections(3)) == Z
```

Comment:
why not keep the test fo this function https://github.com/JuliaReach/LazySets.jl/pull/2298/files#diff-eb4fb67940b1678df4462dad2b59a25cR1691

Reply:
You are right, I will undo the change